python/mkdocs/docs/verify-release.md

```diff
@@ -101,6 +101,6 @@ This will include a Minio S3 container being spun up.
 
 Votes are cast by replying to the release candidate announcement email on the dev mailing list with either `+1`, `0`, or `-1`.
 
-> \[ \] +1 Release this as Apache Iceberg 1.1.0 \[ \] +0 \[ \] -1 Do not release this because…
+> \[ \] +1 Release this as Apache Iceberg 1.2.0 \[ \] +0 \[ \] -1 Do not release this because…
```

Comment:
looks like this sentence itself is not right, since it should be `pyiceberg x.x.x`?